wiki/Ranking_Criteria/skinning/en.md

```diff
@@ -0,0 +1,1002 @@
+		<td dir="ltr">required</td>
+		<td>-</td>
+		<td dir="ltr" rowspan="4">Yes</td>
+	</tr>
+	<tr>
+		<td>cursortrail.png</td>
+		<td dir="ltr">required</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<td>cursormiddle.png</td>
+		<td>should be optional depending on the cursor's design</td>
+		<td>optional</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<td>cursor-smoke.png</td>
+		<td>-</td>
+		<td>optional</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<th rowspan="13">osu!<br>
+		Hitburst Set</td>
+		<td>hit0(-n).png</td>
+		<td rowspan="7">each hitburst should be clearly distinguishable from each other</td>
+		<td dir="ltr">required</td>
+		<td>-</td>
+		<td dir="ltr" rowspan="13">Yes</td>
+	</tr>
+	<tr>
+		<td>hit50(-n).png</td>
+		<td dir="ltr">required</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<td>hit100(-n).png</td>
+		<td dir="ltr">required</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<td>hit100k(-n).png</td>
+		<td dir="ltr">required</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<td>hit300(-n).png</td>
+		<td dir="ltr">required</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<td>hit300g(-n).png</td>
+		<td dir="ltr">required</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<td>hit300k(-n).png</td>
+		<td dir="ltr">required</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<td>lighting.png</td>
+		<td>-</td>
+		<td>optional</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<td>particle50.png</td>
+		<td rowspan="3">will only work if custom hitbursts are inlcuded</td>
+		<td>optional</td>
+		<td rowspan="3">7x7</td>
+	</tr>
+	<tr>
+		<td>particle100.ong</td>
+		<td>optional</td>
+	</tr>
+	<tr>
+		<td>particle300.png</td>
+		<td>optional</td>
+	</tr>
+	<tr>
+		<td dir="ltr">sliderpoint10.png</td>
+		<td dir="ltr" rowspan="2">will only work if an user-specific skin supports skin version 1</td>
+		<td dir="ltr">optional</td>
+		<td dir="ltr">-</td>
+	</tr>
+	<tr>
+		<td dir="ltr">sliderpoint30.png</td>
+		<td dir="ltr">optional</td>
+		<td dir="ltr">-</td>
+	</tr>
+	<tr>
+		<th dir="ltr" rowspan="10">osu!<br>
+		Hitcircle Set</td>
+		<td>approachcircle.png</td>
+		<td dir="ltr">-</td>
+		<td dir="ltr">required</td>
+		<td dir="ltr">126x126</td>
+		<td dir="ltr" rowspan="10">Yes</td>
+	</tr>
+	<tr>
+		<td>followpoint.png</td>
+		<td>-</td>
+		<td dir="ltr">required</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<td>hitcircle.png</td>
+		<td dir="ltr" rowspan="2">should always be inlcuded together, used for slidercircles when they are not skinned, won't be used for slidercircles if an user-specific skin has them skinned</td>
+		<td dir="ltr">required</td>
+		<td dir="ltr" rowspan="2">118x118 (circle)<br>
+		128x128 (whole file)</td>
+	</tr>
+	<tr>
+		<td>hitcircleoverlay(-n).png</td>
+		<td dir="ltr">required</td>
+	</tr>
+	<tr>
+		<td>reversearrow.png</td>
+		<td>should face right</td>
+		<td dir="ltr">required</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<td>sliderendcircle.png</td>
+		<td dir="ltr" rowspan="4">should be included to guarantee correct display when an user-specific skin has them skinned</td>
```

Comment:
`a`, see previous
There are more of these btw, not going to go through them all.